src/instructlab/configuration.py

```diff
@@ -286,20 +291,35 @@ class _chat(BaseModel):
 
     # model configuration
     model_config = ConfigDict(extra="ignore")
-
-    # required fields
-    model: str
-
+    model: str = Field(
+        default_factory=lambda: DEFAULTS.DEFAULT_MODEL,
+        description="Model to be used for chatting with.",
+    )
     # additional fields with defaults
-    vi_mode: bool = False
-    visible_overflow: bool = True
-    context: str = "default"
-    session: typing.Optional[str] = None
+    vi_mode: bool = Field(default=False, description="Enable visual mode for chat.")
```

Comment:
Is this visual mode? I was under the impression that this enables vim keybindings.

Reply:
That's right! Thanks!